misc/lint/trailing-newline.sh

```diff
@@ -30,6 +31,9 @@ for file in "$@"; do
     last_byte=$(tail -c1 "$file")
     if [[ "$last_byte" != $'\n' && "$last_byte" != "" ]] &> /dev/null; then
         echo "lint: $(red error:) trailing-newline: $file is missing a trailing newline" >&2
+        errors++
+    fi
+    if [[ $errors -gt 0 ]]; then
         exit 1
     fi
```

Comment:
Doesn't this need to go outside the loop?

Reply:
Yeah, I totally thought I had put it outside the loop. 🤦